### PR TITLE
Update Groovy syntax from using the set() function

### PR DESCRIPTION
**Pull request by sarahhaggarty**
This PR updates any instances in our docs where Groovy syntax uses the `set()` function and replaces it with an assignment operator.

### CODE REVIEW

File: docs/topics/gradle/gradle-compilation-and-caches.md [hunk 1]
```diff
@@ -255,7 +255,7 @@ tasks.withType<CompileUsingKotlinDaemon>().configureEach {
 
 ```groovy
 tasks.withType(CompileUsingKotlinDaemon::class).configureEach { task ->
```
Comment (ALikhachev) [inline docs/topics/gradle/gradle-compilation-and-caches.md:257]:
```suggestion
tasks.withType(CompileUsingKotlinDaemon).configureEach { task ->
```
Oh, we had this sample incorrect. This is a correction to make it work.

File: docs/topics/gradle/gradle-configure-project.md [hunk 2]
```diff
@@ -317,7 +317,7 @@ kotlin {
 ```groovy
 kotlin {
     jvmToolchain {
-        languageVersion.set(JavaLanguageVersion.of(<MAJOR_JDK_VERSION>))
+        languageVersion = JavaLanguageVersion.of(<MAJOR_JDK_VERSION>))
```
Comment (ALikhachev) [inline docs/topics/gradle/gradle-configure-project.md:320]:
```suggestion
        languageVersion = JavaLanguageVersion.of(<MAJOR_JDK_VERSION>)
```

**Review by ALikhachev — CHANGES_REQUESTED** [review]
Looking at these samples, I see another kind of inconsistency. 

1. In some samples we add import statements to reference classes by short names. (like here https://github.com/JetBrains/kotlin-web-site/blob/56844680abde3eb74de066df6634f2ad19d82074/docs/topics/gradle/gradle-compilation-and-caches.md#L336)
2. In other samples we reference classes by FQN (like here https://github.com/JetBrains/kotlin-web-site/blob/2b6165469b2102b845225b569f1cee1384db6416/docs/topics/gradle/gradle-configure-project.md#L192)
3. The most problematic case is when we reference classes by short names without import statement. (like here https://github.com/JetBrains/kotlin-web-site/blob/56844680abde3eb74de066df6634f2ad19d82074/docs/topics/gradle/gradle-compilation-and-caches.md#L248)

I think it should be addressed separately anyway

**Comment by sarahhaggarty** [comment]
> Looking at these samples, I see another kind of inconsistency.
> 
> 1. In some samples we add import statements to reference classes by short names. (like here https://github.com/JetBrains/kotlin-web-site/blob/56844680abde3eb74de066df6634f2ad19d82074/docs/topics/gradle/gradle-compilation-and-caches.md#L336
>    )
> 2. In other samples we reference classes by FQN (like here https://github.com/JetBrains/kotlin-web-site/blob/2b6165469b2102b845225b569f1cee1384db6416/docs/topics/gradle/gradle-configure-project.md#L192
>    )
> 3. The most problematic case is when we reference classes by short names without import statement. (like here https://github.com/JetBrains/kotlin-web-site/blob/56844680abde3eb74de066df6634f2ad19d82074/docs/topics/gradle/gradle-compilation-and-caches.md#L248
>    ) When one just copy-pastes such samples, they do not work without additional actions
> 
> I think this inconsistency should be addressed separately anyway

Thanks for pointing this out! I'll fix it in another PR :)

**Review by ALikhachev — APPROVED** [review]

**Pull request merged**